src/core/src/service_interfaces/TelemetryWriter.py

```diff
@@ -87,35 +76,16 @@ def send_machine_config_info(self):
             'machine_arch': str(self.env_layer.platform.machine()),
             'disk_type': self.get_disk_type()
         }
-        return self.send_config_info(machine_info, 'machine_config')
+        return self.write_config_info(machine_info, 'machine_config')
 
-    def send_execution_error(self, cmd, code, output):
+    def write_execution_error(self, cmd, code, output):
         # Expected to log any errors from a cmd execution, including package manager execution errors
         error_payload = {
             'cmd': str(cmd),
             'code': str(code),
             'output': str(output)[0:3072]
```

Comment:
Note that this is a string truncation to avoid excessive payload sizes in V1. Don't leave this as an artefact behind - you've got mechanisms for this, so have it centralized.